OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -599,7 +599,7 @@ public int MovementSpeedForCell(Actor self, CPos cell)
 				return 0;
 
 			speed *= Info.Speed;
-			foreach (var t in speedModifiers)
+			foreach (var t in speedModifiers.Value)
```

Comment:
As we discussed in IRC we should clean up the easy `// TODO` six lines above while here.